src/main/java/Interactors/TextFileTranslator.java

```diff
@@ -31,33 +36,74 @@ public File[] checkSaves(String folderPath){
     }
 
     @Override
-    public ArrayList<String[]> loadGame(String filePath) throws FileNotFoundException {
+    public ArrayList<ArrayList<String[]>> loadGame(String filePath) throws FileNotFoundException {
         // loadGame reads the given filePath and returns an ArrayList of String arrays
         // each new line on in the txt file given by filePath contains the instance attributes of a Player instance
         // the Board instance attributes are separated from the Player instance attributes by a header "Board"
         // the Board instance attributes are stored in the last element of the ArrayList
 
-        ArrayList<String[]> gameInfo = new ArrayList<>();
+        ArrayList<ArrayList<String[]>> gameInfo = new ArrayList<>();
+        ArrayList<String[]> players = new ArrayList<>();
+        ArrayList<String[]> playerPositions = new ArrayList<>();
+        ArrayList<String[]> savedTree = new ArrayList<>();
+        ArrayList<String[]> treeOptions = new ArrayList<>();
+        boolean player = false;
         boolean board = false;
+        boolean tree = false;
+        boolean options = false;
 
         File gameData = new File(filePath);
         Scanner scan = new Scanner(gameData);
 
         while (scan.hasNextLine()) {
             String data = scan.nextLine();
-            if (data.trim().equals("Board")) {
-                board = true;
-                continue;
+
+            switch (data.trim()) {
+                case "playerStart":
+                    player = true;
+                case "playerEnd":
+                    player = false;
+                    break;
+                case "positionStart":
+                    board = true;
+                    break;
+                case "positionEnd":
+                    board = false;
+                    break;
+                case "treeStart":
+                    tree = true;
+                    break;
+                case "treeEnd":
+                    tree = false;
+                    break;
+                case "optionStart":
+                    options = true;
+                    break;
+                case "optionEnd":
+                    options = false;
+                    break;
             }
```

Comment:
We can likely cut down the switch block in half by having some switches occur at the same time. However, this is slightly easier to read, so we might not want to do that.

Reply:
Good use of the switch statements compared to the if/else statements